apps/dashboard/src/app/(dashboard)/event/[id]/page.tsx

```diff
@@ -7,6 +7,7 @@ import { useEventDetailsContext } from "./provider"
 import { EventCompaniesPage } from "./companies-page"
 import { EventEditCard } from "./edit-card"
 import { useRouter } from "next/navigation"
+import { EventDetailsAttendance } from "./details-attendance"
```

Comment:
small nit, rename to attendance-page to match the other tabs

Reply:
how so? both the file and import seem to match the naming convention?

Reply:
I was thinking of this one https://github.com/dotkom/monoweb/blob/main/apps/dashboard/src/app/(dashboard)/event/%5Bid%5D/companies-page.tsx

renaming details-attendance to EventAttendancePage